content/examples/flatcar-container-linux/_index.md

```diff
@@ -0,0 +1,204 @@
++++
+title = "Flatcar Container Linux"
+date = 2020-05-11T06:50:00+00:00
+draft = false
+weight = 10
+toc = true
+
++++
+
+This document describes the process of provisioning Flatcar Container Linux machines using Tinkerbell.
+
+## Preparing flatcar-install Docker image
+
+To install [Flatcar Container Linux](https://www.flatcar-linux.org/) as the target OS on the worker, the Docker image from the following `Dockerfile` needs to be pushed into the Tinkerbell registry:
+
+```
+FROM ubuntu
+
+RUN apt update && apt install -y udev gpg wget
+
+RUN wget https://raw.githubusercontent.com/flatcar-linux/init/flatcar-master/bin/flatcar-install -O /usr/local/bin/flatcar-install && \
+    chmod +x /usr/local/bin/flatcar-install
+```
+
+Before you build the image, make sure you have `TINKERBELL_HOST_IP` environment variable set, pointing to the Tinkerbell server IP address. This can usually be done by sourcing `envrc`file, generated by Tinkerbell's `setup.sh` script. To load the environment variables from this file, run the following command:
```

Comment:
```suggestion
Before you build the image, make sure you have `TINKERBELL_HOST_IP` environment variable set, pointing to the Tinkerbell server IP address. This can usually be done by sourcing `envrc` file, generated by Tinkerbell's `setup.sh` script. To load the environment variables from this file, run the following command:
```